src/config/mongoose_config_parser_toml.erl

```diff
@@ -91,14 +91,6 @@ process_section([<<"services">>] = Path, Content) ->
 process_section([<<"modules">>] = Path, Content) ->
     Mods = parse_section(Path, Content),
     ?HOST_F([#local_config{key = {modules, Host}, value = Mods}]);
-process_section([<<"s2s">>] = Path, Content) ->
```

Comment:
At first sight, I can see that `process_section` has clauses for many other sections, but not now for `s2s`, isn't that a bit suspicious? 😕

Reply:
This is because the `s2s` section is now handled by `parse_section`, just like `shaper`, `acl` or `access`. These clauses are more like exceptions than the norm and they should disappear in the future when we get rid of the nested tuples in the internal configuration format.